example/bookstore/v1/bookstore.pb.bookstore_service.oas31.yaml

```diff
@@ -271,8 +289,9 @@ paths:
                 $ref: '#/components/schemas/bookstore.v1.ListShelvesResponse'
           description: |2
              Response to ListShelves call.
+      summary: List Shelves
       tags:
-        - Bookstore
+        - Shelves
```

Comment:
Why are these tags changing?

Reply:
Do we have to change the output of `x-speakeasy-group` to match?

Reply:
No, I purposefully left the speakeasy group extension since I'm not sure how its being used.